recipe/api-genrm/run_func_rm.sh

```diff
@@ -0,0 +1,36 @@
+set -x
```

Comment:
Could you show the training results and paste it to here https://github.com/volcengine/verl/blob/main/docs/algo/baseline.md